addons/website_sale/data/product_snippet_template_data.xml

```diff
@@ -435,7 +435,7 @@
             </t>
         </template>
         <template id="price_dynamic_filter_template_product_product" name="Dynamic Product Filter Price">
-            <t t-if="data['prevent_zero_price_sale']">
+            <t t-if="data['prevent_zero_price_sale'] or not website.prevent_zero_price_sale">
```

Comment:
This also seems wrong
can you explain why you need to change this?

Reply:
When we add an alternative product, we do not get the price of that alternative product. It always shows that the product is "Not Available For Sale". To get the price of the product, I had to add the condition for that.

Reply:
Still your condition is wrong :(

Reply:
I have made some changes.
Now there is no need to add that condition.